judge/migrations/0058_problem_currator_field.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Kirito Feng on shrooms
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0057_blue_pretests'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='problem',
+            name='curators',
+            field=models.ManyToManyField(Profile, verbose_name=_('curators'), blank=True, related_name='curated_problems',
+                                     help_text=_('These users will be able to edit a problem, but not be publicly shown as an author.'))
```

Comment:
What drugs are you on? You do not use `_` in migrations for obvious reasons. Instead of fixing, you unfixed the other line...